.storybook/preview.js

```diff
@@ -2,12 +2,20 @@ import React from 'react';
 import {INITIAL_VIEWPORTS} from '@storybook/addon-viewport';
 import isChromatic from 'chromatic/isChromatic';
 import {StylingGlobals} from '@twilio-paste/styling-library';
+import {Globals as AnimatedGlobals} from '@twilio-paste/animation-library';
 import {Theme} from '@twilio-paste/theme';
 import {Box} from '@twilio-paste/box';
 import {Stack} from '@twilio-paste/stack';
 import {Grid, Column} from '@twilio-paste/grid';
 import {RenderPerformanceProfiler} from './RenderPerformanceProfiler';
 
+const disableAnimations = isChromatic();
+if (disableAnimations) {
+  AnimatedGlobals.assign({
+    skipAnimation: disableAnimations,
+  });
+}
```

Comment:
question: If theme provider calls skipAnimation internally, but you are also calling it here in the story global context, does that mean `disableAnimations` doesn't actually fully disabled animations?

Reply:
Yeah so the first commit fixed the issue on the Gatsby website (dev and prod), but for some reason the issue still occurs in Chromatic where it loops once before halting.  I'm still debugging.